src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidget.java

```diff
@@ -164,9 +212,8 @@ public void onFailure(Throwable caught) {
 	}
 	
 	@Override
-	public void sendJoinRequest(String message, boolean isAcceptingInvite) {
+	public void sendJoinRequest(String message) {
 		this.message = message;
-		this.isAcceptingInvite = isAcceptingInvite;
```

Comment:
Please test this case:
1.  User 1 invites User 2 to Team A.
2.  User 2 visits wiki page with Join button to Team A.
3.  User 2 clicks Join button.
Result?  Used to show Request Sent message. Does it behave well with this PR?